pkg/controllers/node/termination/controller.go

```diff
@@ -46,6 +48,20 @@ import (
 	nodeclaimutil "sigs.k8s.io/karpenter/pkg/utils/nodeclaim"
 )
 
+var (
```

Comment:
If we go with the backoff approach, where should I put these?

Reply:
I'm thinking we should handle this through the reconciler natively, in which case inlining it is fine.